lib/Backend/Commands.hs

```diff
@@ -176,11 +185,13 @@ deleteFieldCmd (DeleteFieldOptions path fieldName) = do
           let newEntry = entry
                 & fields . at fieldName .~ Nothing
                 & dateModified .~ nowUtc
-          void $ writeSecret newEntry
+          void $ writeSecret backend newEntry
           pure $ DFRSuccess newEntry
 
-findCmd :: (Member BackendEffect r, Member (Error CofferError) r) => FindOptions -> Sem r (Maybe Directory)
-findCmd (FindOptions pathMb textMb sortMb filters filterFields) = do
+findCmd :: (Member BackendEffect r, Member (Error CofferError) r) => Config -> FindOptions -> Sem r (Maybe Directory)
+findCmd config (FindOptions qPathMb textMb sortMb filters filterFields) = do
+  let backendNameMb = maybe ((eitherToMaybe . newBackendName) $ mainBackend config) qpBackendName qPathMb
```

Comment:
No need to default to use `maybe + mainBackend` here. This is already done in `getBackend`.

Reply:
Done